target/linux/ar71xx/base-files/etc/board.d/01_leds

```diff
@@ -320,6 +320,12 @@ rb-lhg-5nd)
 	ucidef_set_led_rssi "rssimediumhigh" "RSSIMEDIUMHIGH" "rb:green:rssi3" "wlan0" "60" "100" "-59" "13"
 	ucidef_set_led_rssi "rssihigh" "RSSIHIGH" "rb:green:rssi4" "wlan0" "80" "100" "-79" "13"
 	;;
+rb-map-2nd)
+	ucidef_set_led_switch "eth1" "WAN" "rb:green:eth1" "switch0" "0x02"
+	ucidef_set_led_switch "eth2" "LAN" "rb:green:eth2" "switch0" "0x04"
```

Comment:
Please, use `ucidef_set_led_netdev` here.

Reply:
not possible: it's a switch.

Reply:
Why?
Netdev does not make sense since ports are connected to internal QCA9531 fast ethernet switch?
With netdev you will always have link on LED

Reply:
Are you guys sitting here all the time, just waiting to reply? :)